crates/hyperqueue/src/client/commands/wait.rs

```diff
@@ -28,14 +29,32 @@ pub async fn wait_for_jobs(
     let response = rpc_call!(
         connection,
         FromClientMessage::WaitForJobs(WaitForJobsRequest {
-            selector,
+            selector: selector.clone(),
         }),
         ToClientMessage::WaitForJobsResponse(r) => r
     )
     .await?;
 
+    let detail = rpc_call!(
+        connection,
+        FromClientMessage::JobDetail(JobDetailRequest {
+            job_id_selector: selector.clone(),
```

Comment:
```suggestion
            job_id_selector: selector,
```
It doesn't matter here, but if it's the last usage within the function, you don't need to clone it and you can just move it.